front-end/src/App.js

```diff
@@ -33,6 +34,14 @@ function App() {
         <Route path="/login">
           <Login />
         </Route>
+        <Route path="/editprofile" render={() => (
+          isLoggedIn() ? (
```

Comment:
Super low priority but look into creating a Wrapper component to protect Routes that require you to be logged in